src/hotspot/share/gc/serial/tenuredGeneration.hpp

```diff
@@ -156,7 +156,11 @@ class TenuredGeneration: public Generation {
 
   virtual void update_gc_stats(Generation* current_generation, bool full);
 
-  virtual bool promotion_attempt_is_safe(size_t max_promoted_in_bytes) const;
+  // Returns true if promotions of the specified amount are
+  // likely to succeed without a promotion failure.
+  // Promotion of the full amount is not guaranteed but
+  // might be attempted in the worst case.
+  bool promotion_attempt_is_safe(size_t max_promoted_in_bytes) const;
```

Comment:
Could you move this declaration up above the `// Performance Counter Support`, so that it doesn't look like this function blongs to the `// Statistics` section?

Reply:
Never mind. I see that it wasn't your change that placed it there.